lib/smart_app_launch/backend_services_group.rb

```diff
@@ -0,0 +1,237 @@
+require_relative 'authorization_request_builder'
+
+module SMARTAppLaunch
+  class SMARTBackendServices < Inferno::TestGroup
+    title 'Backend Services'
+    short_description 'Demonstrate SMART Backend Services Authorization'
+
+    id :smart_backend_services
+
+    input :bulk_token_endpoint,
+          title: 'Backend Services Token Endpoint',
+          description: <<~DESCRIPTION
+            The OAuth 2.0 Token Endpoint used by the Backend Services specification to provide bearer tokens.
+          DESCRIPTION
+    input :bulk_client_id,
+          title: 'Bulk Data Client ID',
+          description: 'Client ID provided at registration to the Inferno application.'
+    input :bulk_scope,
+          title: 'Bulk Data Scopes',
+          description: 'Bulk Data Scopes provided at registration to the Inferno application.',
+          default: 'system/*.read'
+    input :bulk_encryption_method,
+          title: 'Encryption Method',
+          description: <<~DESCRIPTION,
+            The server is required to suport either ES384 or RS384 encryption methods for JWT signature verification.
+            Select which method to use.
+          DESCRIPTION
+          type: 'radio',
+          default: 'ES384',
+          options: {
+            list_options: [
+              {
+                label: 'ES384',
+                value: 'ES384'
+              },
+              {
+                label: 'RS384',
+                value: 'RS384'
+              }
+            ]
+          }
+    input :bulk_jwks_kid,
+          title: 'Bulk Data JWKS kid',
+          description: <<~DESCRIPTION,
+            The key ID of the JWKS private key to use for signing the client assertion when fetching an auth token.
+            Defaults to the first JWK in the list if no kid is supplied.
+          DESCRIPTION
+          optional: true
+    output :bearer_token
+
+    http_client :token_endpoint do
+      url :bulk_token_endpoint
+    end
+
+    test from: :tls_version_test do
+      title 'Authorization service token endpoint secured by transport layer security'
+      description <<~DESCRIPTION
+        [§170.315(g)(10) Test
```

Comment:
This shouldn't contain anything g10 specific. Is this a backend services requirement or just a g10 requirement?

Reply:
This is a backend services requirement too, so I left it, but updated all naming so there isn't anything g10 specific.